src/components/PersonList.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { List, ListItem } from 'react-native-elements';
+import { FlatList } from 'react-native';
+
+export default (props) => (
+  <List>
+    <FlatList
```

Comment:
A list inside a list? 
Listception.
Try to refactor this to use only react-native-elements like [this](https://react-native-training.github.io/react-native-elements/API/lists/#using-rn-listview-implemented-with-link-and-avatar), or use only FlatList.

Reply:
Good point. It's my bad. He copied from a poor example I've shared with him.